src/layout/Flex.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react'
 
-export let Flex = ({
+export default ({
```

Comment:
we should declare this as Flex and then `export default Flex` so that the displayName in the dev tools is `Flex`